packages/plugins/rubygem-http/rubygem-http.spec

```
@@ -1,89 +1,72 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name http
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.3.0
-Release: 2%{?dist}
+Name: rubygem-%{gem_name}
+Version: 5.2.0
+Release: 1%{?dist}
 Summary: HTTP should be easy
-Group: Development/Languages
 License: MIT
 URL: https://github.com/httprb/http
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.2
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(http_parser.rb) >= 0.6.0
-Requires: %{?scl_prefix}rubygem(http_parser.rb) < 0.7
-Requires: %{?scl_prefix}rubygem(http-form_data) >= 2.0
-Requires: %{?scl_prefix}rubygem(http-form_data) < 3
-Requires: %{?scl_prefix}rubygem(http-cookie) >= 1.0
-Requires: %{?scl_prefix}rubygem(http-cookie) < 2
-Requires: %{?scl_prefix}rubygem(addressable) >= 2.3
-Requires: %{?scl_prefix}rubygem(addressable) < 3
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 2.2
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.6
+BuildRequires: ruby >= 2.6
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
+Requires: (rubygem(base64) or ruby-default-gems < 3.4)
+
 %description
 An easy-to-use client library for making requests from Ruby. It uses a simple
 method chaining system for building requests, similar to Python's Requests.
 
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
+%setup -q -n  %{gem_name}-%{version}
 
-%setup -q -D -T -n  %{gem_name}-%{version}
+%gemspec_remove_dep -g base64
 
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+# Replace llhttp-ffi with llhttp because it's more performant and easier to
+# package. We also don't care about non-MRI Ruby implementations.
+# https://github.com/httprb/http/issues/797
+%gemspec_remove_dep -g llhttp-ffi "~> 0.5.0"
+%gemspec_add_dep -g llhttp "~> 0.5.0"
```

the versioning between `lhttp` and `lhttp-ffi` is not in sync, this should be `0.6.0` (as you packaged 0.6.1)

https://github.com/bryanp/llhttp/releases

```suggestion
%gemspec_remove_dep -g llhttp-ffi "~> 0.5.0"
%gemspec_add_dep -g llhttp "~> 0.6.0"
```